spec/models/event_spec.rb

```diff
@@ -70,4 +70,20 @@
       expect(event.for_feed).to eq(expected_json)
     end
   end
+
+  describe '#generic_event_attributes' do
+    it 'returns the correct attributes' do
+      expected = {
+        created_at: nil,
+        created_by: 'unknown',
+        details: {},
+        eventable: be_a(Move),
+        recorded_at: be_a(Time),
+        supplier_id: '1234',
```

Comment:
Maybe can you pass in the `supplier_id` in the event factory, so the spec is less coupled with the factories?